### PR TITLE
FISH-495 Web Profile Certification

### PR DESCRIPTION
**Pull request by MarkWareham**
Publish web profile Jakarta TCK results for filing for Web Profile Certification

### CODE REVIEW

**Review by cubastanley — APPROVED** [review]
💯

**Comment by OndroMih** [comment]
Looks good. I just noticed that both this summary and also the summary for Full profile, which is already published, mention Derby DB. Derby isn't in Payara Server anymore and I believe H2 was used, correct?

File: docs/modules/ROOT/pages/jakartaee-certification/5.2020.6/tck-results-web-5.2020.6.adoc [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+
+As required by the
+https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License],
+following is a summary of the TCK results for releases of Jakarta EE Platform, Web Profile.
+
+**Payara Server 5.2020.6 Web Profile Certification Summary**
+
+- Product Name, Version and download URL (if applicable):
++
+https://payara.fish/download[Payara Server 5.2020.6]
```
Comment (rdebusscher) [inline docs/modules/ROOT/pages/jakartaee-certification/5.2020.6/tck-results-web-5.2020.6.adoc:12]: We need to use the compatibility logo to be correct legally (so other TCK result pages). Also, download link should point community page https://www.payara.fish/downloads/payara-platform-community-edition/

**Review by rdebusscher — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/jakartaee-certification/5.2020.6/tck-results-web-5.2020.6.adoc (hunk 1 above)
Comment (MarkWareham) [reply docs/modules/ROOT/pages/jakartaee-certification/5.2020.6/tck-results-web-5.2020.6.adoc]: Surely you can't use the logo until certified?

Comment (OndroMih) [reply docs/modules/ROOT/pages/jakartaee-certification/5.2020.6/tck-results-web-5.2020.6.adoc]: I think that the product name should also be `Payara Server 5.2020.6 (Web Profile)`, which is used on the download page.

**Comment by MarkWareham** [comment]
> Looks good. I just noticed that both this summary and also the summary for Full profile, which is already published, mention Derby DB. Derby isn't in Payara Server anymore and I believe H2 was used, correct?

I believe Derby is used by the TCK

File: docs/modules/ROOT/pages/jakartaee-certification/5.2020.6/tck-results-web-5.2020.6.adoc [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+
+As required by the
+https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License],
+following is a summary of the TCK results for releases of Jakarta EE Platform, Web Profile.
+
+**Payara Server 5.2020.6 Web Profile Certification Summary**
+
+- Product Name, Version and download URL (if applicable):
++
+https://www.payara.fish/downloads/payara-platform-community-edition/[Payara Server 5.2020.6]
```
Comment (MarkWareham) [inline docs/modules/ROOT/pages/jakartaee-certification/5.2020.6/tck-results-web-5.2020.6.adoc:12]:
```suggestion
https://www.payara.fish/downloads/payara-platform-community-edition/[Payara Server 5.2020.6 (Web Profile)]
```

**Pull request merged**